docs/pywbemclisubcommands.rst

```diff
@@ -45,6 +45,190 @@ Command Details`
+each of the requests in an enumeration sequence.
+
+Since the default for use-pull-ops is ``either``, normally pywbem first tries
+the pull operation and then if that fails, the traditional operation.  That
+is probably the most logical setting for ``use-pull-ops`` unless you are
+specifically testing the use of pull operations.
+
+
+.. _`pywbemcli commands to WBEM Operations`:
+
+pywbemcli commands to WBEM Operations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following table defines which pywbemcli commands are used for the
+corresponding pywbem request operations.
+
+=================================  ==============================================
+WBEM CIM-XML Operation             pywbemcli command-group & subcommand
+=================================  ==============================================
+**Instance Operations:**
+EnumerateInstances                 instance enumerate INSTANCENAME
+EnumerateInstanceNames             instance enumerate INSTANCENAME --name_only
+GetInstance                        instance get INSTANCENAME
+ModifyInstance                     instance modify
+CreateInstance                     instance create
+DeleteInstance                     instance delete INSTANCENAME
+Associators(instance)              instance associators INSTANCENAME
+Associators(class)                 class associators CLASSNAME
+AssociatorNames(instance)          instance associators INSTANCENAME --name_only
+AssociatorNames(class)             class associators CLASSNAME --name_only
+References(instance)               instance references INSTANCENAME
+References(class)                  class references CLASSNAME
+ReferenceNames(instance)           instance references INSTANCENAME --name_only
+ReferenceNames(class)              class references CLASSNAME --name_only
+InvokeMethod                       instance invokemethod INSTANCENAME --name_only
+ReferenceNames                     class invokemethod CLASSNAME --name_only
+ExecQuery                          instance query
+**Pull Operations:**               Option --use-pull-ops ``either`` or ``yes``
+OpenEnumerateInstances             instance enumerate INSTANCENAME
+OpenEnumerateInstancePaths         instance enumerate INSTANCENAME --name_only
+OpenAssociatorInstances            instance associators INSTANCENAME
+OpenAssociatorInstancePaths        instance associators INSTANCENAME --name_only
+OpenReferenceInstances             instance references INSTANCENAME
+OpenReferenceInstancePaths         instance references INSTANCENAME --name_only
+OpenQueryInstances                 instance references INSTANCENAME --name_only
+PullInstancesWithPath              part of pull sequence
+PullInstancePaths                  part of pull sequence
+PullInstances                      part of pull sequence
+CloseEnumeration                   Not implemented
+**Class Operations:**
+EnumerateClasses                   class enumerate CLASSNAME
+EnumerateClassNames                class enumerate --name-only
```

Comment:
Update option name `--name-only` to `--names-only`.

Reply:
DONE